[2025-07-07T07:51:17.468+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dim_policy.run_etl_dim_policy manual__2025-07-07T07:46:08.873391+00:00 [queued]>
[2025-07-07T07:51:17.482+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dim_policy.run_etl_dim_policy manual__2025-07-07T07:46:08.873391+00:00 [queued]>
[2025-07-07T07:51:17.483+0000] {taskinstance.py:1308} INFO - Starting attempt 2 of 2
[2025-07-07T07:51:17.493+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): run_etl_dim_policy> on 2025-07-07 07:46:08.873391+00:00
[2025-07-07T07:51:17.499+0000] {standard_task_runner.py:57} INFO - Started process 635 to run task
[2025-07-07T07:51:17.507+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'etl_dim_policy', 'run_etl_dim_policy', 'manual__2025-07-07T07:46:08.873391+00:00', '--job-id', '240', '--raw', '--subdir', 'DAGS_FOLDER/etl_dim_policy.py', '--cfg-path', '/tmp/tmp218j3lgh']
[2025-07-07T07:51:17.510+0000] {standard_task_runner.py:85} INFO - Job 240: Subtask run_etl_dim_policy
[2025-07-07T07:51:17.625+0000] {task_command.py:410} INFO - Running <TaskInstance: etl_dim_policy.run_etl_dim_policy manual__2025-07-07T07:46:08.873391+00:00 [running]> on host 1622b3ff66d0
[2025-07-07T07:51:17.784+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='etl_dim_policy' AIRFLOW_CTX_TASK_ID='run_etl_dim_policy' AIRFLOW_CTX_EXECUTION_DATE='2025-07-07T07:46:08.873391+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-07T07:46:08.873391+00:00'
[2025-07-07T07:51:17.785+0000] {logging_mixin.py:150} INFO - Starting ETL process...
[2025-07-07T07:51:17.822+0000] {base.py:73} INFO - Using connection ID '***_insurance_bi' for task execution.
[2025-07-07T07:51:19.003+0000] {logging_mixin.py:150} INFO - Extracted 200000 records
[2025-07-07T07:51:19.109+0000] {logging_mixin.py:150} INFO - Transformed 200000 records
[2025-07-07T07:51:19.118+0000] {base.py:73} INFO - Using connection ID '***_insurance_dw' for task execution.
[2025-07-07T07:51:21.887+0000] {logging_mixin.py:150} INFO - ETL failed: operator does not exist: character varying <> integer
LINE 6:                     product_id != 746663 OR
                                       ^
HINT:  No operator matches the given name and argument types. You might need to add explicit type casts.
[2025-07-07T07:51:21.888+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/etl_dim_policy.py", line 133, in etl_dim_policy
    result = load_dim_policy(transformed_df)
  File "/opt/airflow/dags/etl_dim_policy.py", line 82, in load_dim_policy
    (row['policy_id'], row['customer_id'], row['product_id'], row['tanggal_mulai'], row['tanggal_berakhir'])
psycopg2.errors.UndefinedFunction: operator does not exist: character varying <> integer
LINE 6:                     product_id != 746663 OR
                                       ^
HINT:  No operator matches the given name and argument types. You might need to add explicit type casts.

[2025-07-07T07:51:21.913+0000] {taskinstance.py:1350} INFO - Marking task as FAILED. dag_id=etl_dim_policy, task_id=run_etl_dim_policy, execution_date=20250707T074608, start_date=20250707T075117, end_date=20250707T075121
[2025-07-07T07:51:21.928+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 240 for task run_etl_dim_policy (operator does not exist: character varying <> integer
LINE 6:                     product_id != 746663 OR
                                       ^
HINT:  No operator matches the given name and argument types. You might need to add explicit type casts.
; 635)
[2025-07-07T07:51:21.971+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2025-07-07T07:51:22.005+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
